pkg/operator/api/userconfig/apis.go

```diff
@@ -31,19 +31,24 @@ type APIs []*API
 
 type API struct {
 	ResourceFields
-	Model          string      `json:"model" yaml:"model"`
-	ModelFormat    ModelFormat `json:"model_format" yaml:"model_format"`
-	Tracker        *Tracker    `json:"tracker" yaml:"tracker"`
-	RequestHandler *string     `json:"request_handler" yaml:"request_handler"`
-	Compute        *APICompute `json:"compute" yaml:"compute"`
-	Tags           Tags        `json:"tags" yaml:"tags"`
+	Model          string            `json:"model" yaml:"model"`
+	ModelFormat    ModelFormat       `json:"model_format" yaml:"model_format"`
+	Tracker        *Tracker          `json:"tracker" yaml:"tracker"`
+	RequestHandler *string           `json:"request_handler" yaml:"request_handler"`
+	Compute        *APICompute       `json:"compute" yaml:"compute"`
+	Tags           Tags              `json:"tags" yaml:"tags"`
+	TFServing      *TFServingOptions `json:"tf_serving" yaml:"tf_serving"`
```

Comment:
nit: Can you add this maybe between RequestHandler and Compute, since it kinda fits up there better?